summary_workflows/quantification/quantification_dockers/q_metrics/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Use an official Python runtime as a parent image
-FROM ubuntu:latest
+FROM ubuntu:20.04
```

Comment:
20.04 was the version that was in the container that the workflow built when I ran it, and since it worked I took that :) . If you prefer a different version please go ahead!

Reply:
You're saying that `ubuntu:latest` installed 20.04? I see that there are a handful of newer versions in [Ubuntu's Docker Hub  page](https://hub.docker.com/_/ubuntu?tab=tags) but 20.04 is the latest LTS version, perhaps that's why it was the one installed?

Reply:
Yes, ubuntu:latest installed 20.04! Not an expert here, but I guess having an LTS version wouldn't be wrong. So we're good, right?

Reply:
Sure, all good.

Just for the record, it seems that Ubuntu's `latest` tag always points to the latest LTS version which is the one Ubuntu recommends using:
```
20.04, focal-20220113, focal, latest
(...)
The ubuntu:latest tag points to the "latest LTS", since that's the version recommended for general use. The ubuntu:rolling tag points to the latest release (regardless of LTS status).
```
The next LTS version will be 22.04 but we should be fine using 20.04 for the time being.